static-site/_plugins/copy_assets.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+
+Jekyll::Hooks.register :site, :after_init do ||
+   copy_data_dictionary("1")
+   copy_data_dictionary("2")
+   copy_openapi()
+  end
+
+def copy_data_dictionary(version)
+    contents = find_dist_file("../../dist/V#{version}-data-dictionary*json")
+    # un-escape extra newlines in the example json so it renders properly on the site
+    replaced = contents.gsub(/\\\\n/, "\\n")
+    current_dir = File.dirname(__FILE__)
+    data_folder = File.join(current_dir, "../_data");
+    FileUtils.mkdir_p data_folder
+    dd_dest = File.join(data_folder, "/data-dictionary-v#{version}.json")
+    write_file(dd_dest, replaced)
+end
+
+def copy_openapi()
```

Comment:
added the script to copy the OpenAPI spec here, the rest is just moved from the old `copy_data_dictionary.rb` file.